src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/entity/session/PermittedSessionLocation.kt

```diff
@@ -0,0 +1,56 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.model.entity.session
+
+import org.hibernate.annotations.CreationTimestamp
+import org.hibernate.annotations.UpdateTimestamp
+import org.springframework.data.jpa.repository.Temporal
+import uk.gov.justice.digital.hmpps.visitscheduler.model.entity.Prison
+import java.time.LocalDateTime
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToMany
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+import javax.persistence.TemporalType
+
+@Entity
+@Table(name = "PERMITTED_SESSION_LOCATION")
+data class PermittedSessionLocation(
```

Comment:
This is something for later but  there is a warning that JPA entities should not be data class. So we will have to change all the entities on VSIP defined as data class.

Reply:
agreed